src/components/CreateTodo.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <div class="createTodo card">
+    <div class="createTodo__inputBox">
+      <label for="createTodo__inputBox_titleLabel">Title</label>
+      <input
+        type="text"
+        class="createTodo__inputBox_titleInput"
+        :value="todoTitle"
+        @input="$emit('title-input', $event.target.value)"
+      />
+      <span v-show="isTitleError">Can't add empty title.</span>
+
+      <label for="createTodo__inputBox_descLabel">Description</label>
+      <textarea
+        class="createTodo__inputBox_descInput"
+        :value="todoDesc"
+        @input="$emit('desc-input', $event.target.value)"
+      />
+      <span v-show="isDescError">Can't add empty description.</span>
+    </div>
+
+    <div class="createTodo__button">
+      <button class="createTodo__button_add btn" @click="$emit('add')">
+        Add
+      </button>
+      <div class="createTodo__button_cancel btn" @click="$emit('cancel')">
+        <DeleteIcon />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import DeleteIcon from "@/components/icons/DeleteIcon.vue"
+
+export default {
+  components: {
+    DeleteIcon,
+  },
+  props: {
+    todoTitle: {
+      type: String,
+    },
+    todoDesc: {
+      type: String,
+    },
+    isTitleError: {
+      type: Boolean,
+      default: false,
+    },
+    isDescError: {
+      type: Boolean,
+      default: false,
+    },
+  },
+}
+</script>
+
+<style lang="scss">
+textarea {
+  resize: none;
+  height: 5rem;
+}
```

Comment:
You did not use scope in style and use textarea directly. This will affect other textarea of the application. 

Two things need to do. 
1. use specific class for textarea instead directly select it. 
3. Use scoped in style tag.